apkid/rules/apk/packers.yara

```diff
@@ -218,6 +218,24 @@ rule dexprotector_b : packer
     not dexprotector
 }
 
+rule dexpro_a : packer
```

Comment:
It would be cleaner and easier to review if you added this rule to https://github.com/rednaga/APKiD/pull/199.

Reply:
Done!